content/en/blog/_posts/2025-04-23-jobs-successpolicy-goes-ga.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: blog
+title: "Kubernetes 1.33: Job's SuccessPolicy Goes GA"
+date: 2025-04-23
+slug: kubernetes-1-33-jobs-success-policy-goes-ga
+authors: >
+  [Yuki Iwai](https://github.com/tenzen-y) (CyberAgent, Inc)
+---
+
+Announcing the graduation to General Availability (GA) of the Job's __Success Policy__, in Kubernetes v1.33.
+
+## About Job's Success Policy
+
+When you run batch workloads with leader-worker patterns, you might want to mark it as succeeded
```

Comment:
If you can point to some examples, wiki page it'd be nice. Also the majority of today's literature is using leader-follower name, so I'd suggest using that here as well.

Reply:
Thank you. I will replace `leader-worker` with `leader-follower` in all places.
Additionally, I am trying to explore the wiki for leader-follower pattern.